### ISSUE TITLE
Checkout Lottie to render the coffee drinking animation

### ISSUE THREAD

**Issue by d4rken** [d4rken/sdmaid-public#728]
https://github.com/airbnb/lottie-android

**Comment by SuperSandro2000** [comment]
Does this fix this? Using Swift installer but this looks like this since years on all themes.
![screenshot_20180919-083828_sd maid](https://user-images.githubusercontent.com/7258858/45735272-95ced600-bbe7-11e8-9fcb-de74068a83a7.jpg)

**Comment by d4rken** [comment]
Yes this fixes that. Previously I used a GIF for the animation and GIFs don't support transparency.

Now we have a nice vector animation:

https://www.lottiefiles.com/share/ZLnBJG

**Comment by SuperSandro2000** [comment]
Website does not work on mobile but on desktop you can see the lovely work. Thanks man!
